packages/adp-tooling/templates/rta/controller.ejs

```diff
@@ -0,0 +1,86 @@
+/*!
```

Comment:
Please remove the license statement. If I as a developer create the file using the tool then it should not have SAP copyright.